### ISSUE TITLE
Accessibility and UX

### ISSUE THREAD

**Issue by tassoevan** [RocketChat/Rocket.Chat#12061]
@entonio commented on [Fri Sep 14 2018](https://github.com/RocketChat/Rocket.Chat.Electron/issues/879)

We've moved to Rocket.Chat from Slack and while the experience has been good there are minor accessibility points that end up being serious:
- The messages font is gray. The difference re Slack is noticeable. It's a good deal harder on my eyes.
- The general monochrome approach may be neat to others, but it's tiresome to me in extended usage. Slack has a number of coloured themes, I tend to Aubergine or Monument.
- The space above the message input field is too tall. Most of the times it's empty and just looks to me like a UI margin bug. I understand it's there for 'is typing' notifications, but why not put those below the input field where there is always space and is customary?

Sorry for mixing up issues, and on the whole kudos for a nice app, which is already in some respects superior to the others.

**Comment by tassoevan** [comment]
@ggazzo and @thiagosanchz, any thoughts?
